addons/l10n_om/data/template/account.fiscal.position-om.csv

```diff
@@ -0,0 +1,3 @@
+"id","sequence","name","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","name@ar"
+"fiscal_position_template_dom","1","Oman","True","","base.om","","l10n_om_sale_vat_5","l10n_om_sale_0_ex","عمان"
+"fiscal_position_non_oman","2","Non-Oman","True","","","","l10n_om_purchase_vat_5","l10n_om_purchase_5_ex","غير عمان"
```

Comment:
Not sure the mapping of account is right, we might need to double check with po on that :smile: